### PR TITLE
feat: expose SetLogLevel in cli

### PR DESCRIPTION
**Pull request by iwpnd**
Allow user to set a desired log level on startup with `--log-level Level`.
Omitting the flag results in the default INFO

Hey 👋 

I got to one of my proposition from #831. 
As options I went for the log levels that @gdey has already set - all I did is exposing `SetLogLevel` in the root. 

Let me know if you prefer those levels for the flag instead:

```
very-verbose = TRACE
verbose = DEBUG
default = INFO
quiet = WARN
silent = ERROR
```

Where do the docs live actually? 🤔

### CODE REVIEW

File: cmd/tegola/cmd/root.go [hunk 1]
```diff
@@ -22,14 +23,34 @@ var (
...
+	default:
+		return 0, fmt.Errorf("invalid log level use")
+	}
+}
+
 func init() {
 	// root
 	RootCmd.PersistentFlags().StringVar(&configFile, "config", "config.toml",
 		"path or http url to a config file, or \"-\" for stdin")
+	RootCmd.PersistentFlags().StringVar(&logLevel, "log-level", "INFO",
```
Comment (ARolek) [inline cmd/tegola/cmd/root.go:47]: we should probably put the various options here in the long description.

**Review by ARolek — COMMENTED** [review]
This looks fine to me. Since tegola is not using the centralized log package throughout the codebase yet, should we add a note to the flag to say it's only partially adopted at this point?

**Comment by iwpnd** [comment]
understood. I added the options and labelled the flag as a work in progress.

**Comment by iwpnd** [comment]
@gdey since this is actually your work and not mine, are you okay with this?

**Comment by ARolek** [comment]
closed in https://github.com/go-spatial/tegola/commit/e593b4f8518ea2c399834da0048a50261dce9fe5

**Pull request closed without merging**